MessageManager.class.nut

```diff
@@ -792,8 +797,10 @@ class MessageManager {
             _isFunc(msg._onAck) && msg._onAck(msg);
             _isFunc(_onAck) && _onAck(msg);
 
-            // Delete the acked message from the queue
-            delete _sentQueue[id];
+            // Delete the acked message from the queue if there is no _onReply handler set (either global or message-specific)
```

Comment:
This part seems wrong to me. If an onReply handler is set, but the message has never been replied, it will sit in the queue forever. Right?